TypeCobol/Compiler/Types/BuiltinTypes.cs

```diff
@@ -0,0 +1,168 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using TypeCobol.Compiler.Symbols;
+using static TypeCobol.Compiler.Types.Type;
+
+namespace TypeCobol.Compiler.Types
+{
+    /// <summary>
+    /// Some Builtin Types
+    /// </summary>
+    public class BuiltinTypes
+    {
+        public static readonly Type Comp1Type;
+        public static readonly Type Comp2Type;
+        public static readonly Type FloatType;
+        public static readonly Type DoubleType;
+        public static readonly Type PointerType;
+        public static readonly Type FunctionPointerType;
+        public static readonly Type ProcedurePointerType;
+        public static readonly Type IndexType;
+        public static readonly Type ObjectReferenceType;
+
+        public static readonly Type OmittedType;
+        public static readonly Type AlphabeticType;
+        public static readonly Type NumericType;
+        public static readonly Type NumericEditedType;
+        public static readonly Type AlphanumericType;
+        public static readonly Type AlphanumericEditedType;
+        public static readonly Type DBCSType;
+        public static readonly Type FloatingPointType;
+        public static readonly Type OccursType;
+
+        public static readonly Type BooleanType;
+        public static readonly Type DateType;
+        public static readonly Type CurrencyType;
+        public static readonly Type StringType;
+
+
+        #region BuiltinUsageTypes
+        //Built-in Basic usage types.
+        public readonly static Type UsageNoType = new Type(Tags.Usage, UsageFormat.None);
+        public readonly static Type UsageCompType = new Type(Tags.Usage, UsageFormat.Comp);
+        public readonly static Type UsageComp4Type = new Type(Tags.Usage, UsageFormat.Comp4);
+        public readonly static Type UsageComp5Type = new Type(Tags.Usage, UsageFormat.Comp5);
+        public readonly static Type UsageDisplay1Type = new Type(Tags.Usage, UsageFormat.Display1);
+        public readonly static Type UsageNationalType = new Type(Tags.Usage, UsageFormat.National);
+        public readonly static Type UsageBinaryType = new Type(Tags.Usage, UsageFormat.Binary);
+        public readonly static Type UsageComp1Type = new Type(Tags.Usage, UsageFormat.Comp1);
```

Comment:
What is the difference between Comp1Type and UsageComp1Type  ?

Reply:
You are right this is exactly the same type.